third_party/ruby/ruby.BUILD

```diff
@@ -10,7 +10,6 @@ ruby(
         "--localstatedir=/var",
         "--disable-maintainer-mode",
         "--disable-dependency-tracking",
-        "--disable-jit-support",
```

Comment:
The history here is that we put this in so that any version of Ruby running at Stripe would not accidentally have JIT support turned on and therefore it would not accidentally be enabled in prod, etc.

I can totally understand wanting to turn this off for experimental purposes, but I think we should make this configurable on a per-Ruby version basis or we should land this PR as an "enable Ruby 3.2 + YJIT to potentially be built" without this bit, and then leave the JIT enabling for experimental PRs.